### PR TITLE
Add ignore.txt for 2.12

### PR DESCRIPTION
**Pull request by Andersson007**
##### SUMMARY

https://github.com/ansible-collections/overview/issues/45#issuecomment-813821100

### CODE REVIEW

File: changelogs/fragments/147-add_ignore_txt.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+---
+trivial:
+- Add ingore-2.12.txt.
```
Comment (Andersson007) [inline changelogs/fragments/147-add_ignore_txt.yml:3]:
```suggestion
  - Add ingore-2.12.txt.
```

**Comment by Andersson007** [comment]
```
ERROR: plugins/modules/vyos_bgp_global.py:0:0: no-log-needed: Argument 'password' in argument_spec could be a secret, though doesn't have `no_log` set found in config -> neighbor
ERROR: plugins/modules/vyos_ospf_interfaces.py:0:0: no-log-needed: Argument 'key' in argument_spec could be a secret, though doesn't have `no_log` set found in config -> address_family -> authentication -> md5_key
ERROR: plugins/modules/vyos_ospf_interfaces.py:0:0: no-log-needed: Argument 'md5_key' in argument_spec could be a secret, though doesn't have `no_log` set found in config -> address_family -> authentication
ERROR: plugins/modules/vyos_ospf_interfaces.py:0:0: no-log-needed: Argument 'plaintext_password' in argument_spec could be a secret, though doesn't have `no_log` set found in config -> address_family -> authentication
ERROR: plugins/modules/vyos_ospfv2.py:0:0: no-log-needed: Argument 'md5_key' in argument_spec could be a secret, though doesn't have `no_log` set found in config -> areas -> virtual_link -> authentication -> md5
ERROR: plugins/modules/vyos_ospfv2.py:0:0: no-log-needed: Argument 'plaintext_password' in argument_spec could be a secret, though doesn't have `no_log` set found in config -> areas -> virtual_link -> authentication
```

**Comment by Andersson007** [comment]
```
   7 #############################################
   8 #                WARNING                    #
   9 #############################################
  10 #
  11 # This file is auto generated by the resource
  12 #   module builder playbook.
  13 #
  14 # Do not edit this file manually.
  15 #
  16 # Changes to this file will be over written
  17 #   by the resource module builder.
  18 #
  19 # Changes should be made in the model used to
  20 #   generate this file or in the resource module
  21 #   builder template.
  22 #
  23 #############################################
```
so we couldn't fix it here as written in the modules

File: changelogs/fragments/147-add_ignore_txt.yml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+---
+trivial:
+  - Add ingore-2.12.txt.
```
Comment (Andersson007) [inline changelogs/fragments/147-add_ignore_txt.yml:3]:
```suggestion
  - Add ignore-2.12.txt.
```

**Comment by GomathiselviS** [comment]
Hi @Andersson007 , you can copy the ignore-2.11.txt to ignore-2.12.txt to avoid the above sanity issues.

**Comment by Andersson007** [comment]
recheck

**Comment by Andersson007** [comment]
@GomathiselviS can we merge this?

**Comment by Andersson007** [comment]
So as it's green now, I added `gate` not to forget later. It should be fine from the context

**Pull request merged**